mcux/devices/MK64F12/MK64F12_features.h

```diff
@@ -392,6 +392,8 @@
 #define FSL_FEATURE_ENET_HAS_RECEIVE_STATUS_THRESHOLD (1)
 /* @brief Has Extend MDIO Support. */
 #define FSL_FEATURE_ENET_HAS_EXTEND_MDIO (0)
+/* @brief Queue Size for each instance. */
```

Comment:
Should we not just resync these files with SDK 2.9?

Reply:
I feel we can do a separate PR to update all platforms to SDK 2.9

Reply:
I agree a separate PR, but seems like doing that first might be better and easier to see what's going on and keeping things in sync.  @MaureenHelm thoughts?

Reply:
I am going with this approach for now as there is a request to get RT1170 support into the mainline at the earliest. There is a team waiting to add more features.

Reply:
I will work on a separate PR to update to SDK 2.9

Reply:
Initial RT1170 support has been stuck for a long time, so I'd like to get that unstuck and then come back and do a full SDK 2.9 update for all SoCs afterward.